server/models/user.model.js

```diff
@@ -1,29 +1,33 @@
-/**
- * User Schema
- */
-module.exports = (sequelize, DataTypes) => {
-    const User = sequelize.define('User', {
-        id: {
-            type: DataTypes.INTEGER,
-            autoIncrement: true,
-            primaryKey: true,
+import mongoose from 'mongoose';
+
+const UserSchema = new mongoose.Schema({
+    uuid: {
+        type: String,
+        required: true,
+        unique: true,
+        index: true,
+    },
+    username: {
+        type: String,
+        required: true,
+    },
+    devices: [new mongoose.Schema({
+        type: {
+            type: String,
+            enum: ['iOS', 'android'],
+            required: true,
         },
-        uuid: {
-            type: DataTypes.UUID,
-            allowNull: false,
-            unique: true,
+        token: {
+            type: String,
+            required: true,
         },
-        username: {
-            type: DataTypes.STRING,
-            allowNull: false,
+        status: {
+            type: String,
+            enum: ['enabled', 'disabled'],
+            required: true,
+            default: 'enabled',
         },
-    }, {
-        indexes: [
-            {
-                fields: ['uuid'],
-            },
-        ],
-    });
-
-    return User;
-};
+    })],
+});
+export const userModelName = 'User';
+export const User = mongoose.model(userModelName, UserSchema);
```

Comment:
I thought I say in the mongoose docs the recommendation to call `mongoose.model('Model')` in files where you want to use the model, so thats what I'm doing currently, but I can't find that recommendation now.
It would be a bit nicer to just use the `User` object exported here.